File: app/src/main/java/com/example/administrator/encyclopediamvp/Bean/LuckyQQBean.java
package com.example.administrator.encyclopediamvp.Bean;

/**
 * Created by Nixo on 2018/5/17.
 */

public class LuckyQQBean {


    /**
     * showapi_res_error :
     * showapi_res_code : 0
     * showapi_res_body : {"ret_code":0,"desc":"[要面包不要爱情型]，其具体表现为：责任心重，尤其对工作充满热诚，是个彻头彻尾工作狂。但往往因为过分专注职务，而忽略身边的家人及朋友，是个宁要面包不需要爱情的理性主义者。","score":"35分","grade":"凶","analysis":"见异思迁，十九不成，徒劳无功，不如更换"}
     */

    private String showapi_res_error;
    private int showapi_res_code;
    private ShowapiResBodyBean showapi_res_body;

    public String getShowapi_res_error() {
        return showapi_res_error;
    }

    public void setShowapi_res_error(String showapi_res_error) {
        this.showapi_res_error = showapi_res_error;
    }

    public int getShowapi_res_code() {
        return showapi_res_code;
    }

    public void setShowapi_res_code(int showapi_res_code) {
        this.showapi_res_code = showapi_res_code;
    }

    public ShowapiResBodyBean getShowapi_res_body() {
        return showapi_res_body;
    }

    public void setShowapi_res_body(ShowapiResBodyBean showapi_res_body) {
        this.showapi_res_body = showapi_res_body;
    }

    public static class ShowapiResBodyBean {
        /**
         * ret_code : 0
         * desc : [要面包不要爱情型]，其具体表现为：责任心重，尤其对工作充满热诚，是个彻头彻尾工作狂。但往往因为过分专注职务，而忽略身边的家人及朋友，是个宁要面包不需要爱情的理性主义者。
         * score : 35分
         * grade : 凶
         * analysis : 见异思迁，十九不成，徒劳无功，不如更换
         */

        private int ret_code;
        private String desc;
        private String score;
        private String grade;
        private String analysis;

        public int getRet_code() {
            return ret_code;
        }

        public void setRet_code(int ret_code) {
            this.ret_code = ret_code;
        }

        public String getDesc() {
            return desc;
        }

        public void setDesc(String desc) {
            this.desc = desc;
        }

        public String getScore() {
            return score;
        }

        public void setScore(String score) {
            this.score = score;
        }

        public String getGrade() {
            return grade;
        }

        public void setGrade(String grade) {
            this.grade = grade;
        }

        public String getAnalysis() {
            return analysis;
        }

        public void setAnalysis(String analysis) {
            this.analysis = analysis;
        }
    }
}

File: app/src/main/java/com/example/administrator/encyclopediamvp/Bean/LuckyQQBean.kt
package com.example.administrator.encyclopediamvp.Bean

/**
 * Created by Nixo on 2018/5/17.
 */

class LuckyQQBean {


    /**
     * showapi_res_error :
     * showapi_res_code : 0
     * showapi_res_body : {"ret_code":0,"desc":"[要面包不要爱情型]，其具体表现为：责任心重，尤其对工作充满热诚，是个彻头彻尾工作狂。但往往因为过分专注职务，而忽略身边的家人及朋友，是个宁要面包不需要爱情的理性主义者。","score":"35分","grade":"凶","analysis":"见异思迁，十九不成，徒劳无功，不如更换"}
     */

    var showapi_res_error: String? = null
    var showapi_res_code: Int = 0
    var showapi_res_body: ShowapiResBodyBean? = null

    class ShowapiResBodyBean {
        /**
         * ret_code : 0
         * desc : [要面包不要爱情型]，其具体表现为：责任心重，尤其对工作充满热诚，是个彻头彻尾工作狂。但往往因为过分专注职务，而忽略身边的家人及朋友，是个宁要面包不需要爱情的理性主义者。
         * score : 35分
         * grade : 凶
         * analysis : 见异思迁，十九不成，徒劳无功，不如更换
         */

        var ret_code: Int = 0
        var desc: String? = null
        var score: String? = null
        var grade: String? = null
        var analysis: String? = null
    }
}
